Sources/FisherYates/Fisher-Yates_Shuffle.swift

```diff
@@ -8,18 +8,11 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-#if swift(>=2.2) && !swift(>=3.0)
+#if !swift(>=3.0)
```

Comment:
These packages can only be built with Swift 4 (and above) so we don't really need to worry about older Swift versions. Do you mind removing these checks?

Reply:
@NevinBR In that case you might be able to replace the os check with "macOS" as well.

Reply:
Oh! That makes everything way easier!

The fact that these checks were already in the code made me think we were trying to support past versions. But if we aren’t, that’s great! I’ll fix it right away.

Reply:
Thanks!